assets/queries/cloudFormation/memcached_disabled/test/positive2.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Description": "A sample template",
+  "Resources": {
+    "ElasticacheCluster": {
```

Comment:
Change "ElasticacheCluster" to "ElasticacheCluster4"

Reply:
done